client/my-sites/subscribers/components/subscribers-page/subscribers-page-context.tsx

```diff
@@ -90,6 +91,7 @@ export const SubscribersPageProvider = ( {
 	const [ showMigrateSubscribersModal, setShowMigrateSubscribersModal ] = useState( false );
 	const [ debouncedSearchTerm ] = useDebounce( searchTerm, 300 );
 	const { hasManySubscribers } = useManySubsSite( siteId );
+	const { completedJob } = useActiveJobRecognition( siteId ?? 0 );
```

Comment:
I've added `completedJob` to return the data we get from the import endpoint response.